### PR TITLE
Common tag input component

### PR DESCRIPTION
**Pull request by KD1712**
Component implements ControlValueAccessor to synchronize its value with the parent form control.

### CODE REVIEW

File: frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-tag-input/common-tag-input.component.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+//   <app-common-tags-input formControlName="tags"></app-common-tags-input>
+
+@Component({
+  selector: 'app-common-tags-input',
+  templateUrl: './common-tag-input.component.html',
+  styleUrls: ['./common-tag-input.component.scss'],
+  standalone: true,
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
```
Comment (danoswaltCL) [inline frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-tag-input/common-tag-input.component.ts:28]: 😎

**Review by danoswaltCL — COMMENTED** [review]
nice! thank you for taking the time to clean up the implementing code.

File: frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-tag-input/common-tag-input.component.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  writeValue(value: string[]) {
+    this.tags.setValue(value || []);
+  }
+
+  registerOnChange(fn: any) {
+    this.tags.valueChanges.subscribe(fn);
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  registerOnTouched(fn: any) {}
```
Comment (danoswaltCL) [inline frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-tag-input/common-tag-input.component.ts:83]: what's this for? can it be removed?

File: frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-tag-input/common-tag-input.component.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => CommonTagsInputComponent),
+      multi: true,
+    },
+  ],
+  imports: [CommonModule, MatChipsModule, MatFormFieldModule, MatIconModule, MatInputModule, TranslateModule],
+})
+export class CommonTagsInputComponent implements ControlValueAccessor {
+  isChipSelectable = true;
```
Comment (danoswaltCL) [inline frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-tag-input/common-tag-input.component.ts:36]: i think we want isChipSelectable `false`, selecting won't do anything in this context.

File: frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-tag-input/common-tag-input.component.ts (hunk 2 above)
Comment (KD1712) [reply frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-tag-input/common-tag-input.component.ts:83]: `ControlValueAccessor` interface mandates to implement its methods which includes `registerOnTouched`. Currently there is no use for this so i've disabled it

Comment (danoswaltCL) [reply frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-tag-input/common-tag-input.component.ts:83]: ah, ty

**Review by danoswaltCL — APPROVED** [review]

**Pull request merged**